package.json

```diff
@@ -25,7 +25,8 @@
   "dependencies": {
     "@emotion/cache": "^11.7.1",
     "@emotion/react": "^11.9.0",
-    "@emotion/styled": "^11.8.1"
+    "@emotion/styled": "^11.8.1",
+	"libphonenumber-js": "1.10.13"
```

Comment:
Are we using this dependency?

Reply:
No, we are not. Initially I was going to, removing it now.